### DISCUSSION TITLE
Is it possible to provide machine configuration and batch size used for pre-train NeMo ASR model?

### DISCUSSION THREAD

**Discussion by mehadi92** [NVIDIA-NeMo/NeMo#6837] [Q&A] [answered]
Hi, 
[Here](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/asr/results.html#speech-recognition-languages) is the all-available Nemo ASR model. Is it possible to provide machine configuration and batch size corresponding to each model (small, medium, large, xlarge)

It would be very helpful for those who are trying to train  with new data and languages

**Comment by titu1994** [comment, marked as answer]
It's not really tractable cause every used has different gpu with different memory. Providing A100 as a baseline is possible but there are too many variations of model configs to manually test the peak memory and batch size.
